### DISCUSSION TITLE
How does the virtual memory card actually work.

### DISCUSSION THREAD

**Discussion by keiem** [emukidid/swiss-gc#799] [Q&A] [answered]
Hello, I read about the memcard pro GC on 8bitmods.  There they try to sell it with this feature: ' The MemCard PRO could automatically load the associated VMC, providing a dedicated card for each game with 8 channels of 15 blocks.'

Is this a feature of the memcard pro or is this a native swiss feature?  

Why I ask, some time ago I enbaled the virtual memory card feature in swiss.  The first game a tried asked to format the memcard in slot A.  But the second (and third) game I tried asked also to format slot A.  So I switched back to a normal memory card.  Yesterday I wanted to try super mario eclipse and it said my memcard didn't have enough blocks left.  I searche the internet, but didn't really find a explanation what the virtual memory card feature does.

Makes it a new memory card for each game?  And if so, how many blocks does it take for each virtual memory card?  

I love the idea and the features of the memcard pro gc, but if it can be done natively in swiss, it doesn't worth the money.  In my opinion, they also should have made a variant to be a replacement for SD2SP2 in the sp1 or sp2 slot and without a LCD screen.

**Comment by emukidid** [comment, marked as answer]
Swiss will make a 16MB raw file per slot per region. How recently did you try Virtual Cards in Swiss?

**Comment by keiem** [comment]
Just last week with the latest release.  I checked my sd-card and found indeed 4 raw files.  2 for slot A, eu and usa version, and 2 for slot B eu and USA version.  I tried it again yesterday, and now it seems to work although I left my memorycard in slot A.  I didn't try without, so maybe  it prioritize the actual hardware.  Although, I tried Super Mario Eclipse again and now it didn't complain about to less free blocks.

Another thing I tried is making a backup of my memorycard with gcmm.  I copied the raw file to the saves location of Swiss and renamed it to the swiss name, I think memorycardA.EU.raw, and it seems it takes over my saves.  I only tried Mario Kart, but I had all my earnings, but as I said of course, it's possible swiss prioritizes the actual hardware.
